_posts/2019-01-30-gcal-for-project-management.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Using Google Calendar for Account Management"
+date: 2019-01-30 10:00:00
+categories: ["agile", "project-management"]
+author: "emily"
+---
+
+At [Ombu Labs](https://www.ombulabs.com), we like to split our time between working on our own products, open source code and client projects.
+
+Our own products include everything from [OmbuShop](http://www.ombushop.com/), an e-commerce platform, to [FastRuby.io](https://fastruby.io/), a Ruby on Rails upgrade service. In terms of open source, we recently created [Audit Tool](https://audit.fastruby.io) and are constantly searching for more projects to contribute to. We also work on a variety of interesting client projects, and generally take on two to three of them at a time.
+
+**As an Account Manager, it can get hectic trying to manage all of this. [Google Calendar](https://www.google.com/calendar) can be a serious help.**
+
+<!--more-->
+
+#### Using Multiple Calendars
+
+In order to manage the different types of projects we work on at [Ombu Labs](https://www.ombulabs.com), it can be helpful to use multiple calendars.
+
+1. **General Admin/Personal**. This calendar includes our team's daily [scrum](https://www.scrum.org/resources/what-is-scrum) call, sprint retrospectives and occasional [donut calls](https://www.donut.com). Any general office-related or personal tasks could go here. Need to track your billable hours using a time-tracker? Need to create client invoices? Have a performance review coming up? These could all be scheduled in this calendar.
+
+2. **Ombu Labs Products & Open Source**. In this calendar, I have our team's weekly [OmbuShop](http://www.ombushop.com/) call, [FastRuby.io](https://fastruby.io/) pair programming sessions and some hours that are blocked off to work on open source projects.
+
+3. **Client Projects**. This calendar contains all events relating to our client projects. This includes demo calls, check-ins with clients and pair programming sessions.
```

Comment:
@ercohen14 You can link `pair programming` to https://www.ombulabs.com/blog/agile/pair-programming/joys-and-woes-of-pair-programming.html